configs/config.py

```diff
@@ -0,0 +1,23 @@
+from dataclasses import dataclass
+from pathlib import Path
+from typing import List, Optional
+
+import yaml
+
+
+@dataclass
+class Config:
+    video_directory: Path
+    video_files: Optional[List[Path]]
+    data_directory: Path
+    data_name: Optional[str]
```

Comment:
Optionals should default `None` or else they aren't optional - defaults are sort of a pain in the ass with dataclasses (if you want to use inheritance, which you probably won't need for a config class) but anyway that might look like this (or use the `kw_only=True` arg in the `@dataclass` decorator

```suggestion
    video_directory: Path
    data_directory: Path
    video_files: Optional[List[Path]] = None
    data_name: Optional[str] = None
```